tests/www/apply/test_process.py

```diff
@@ -2614,23 +2626,38 @@ def test_accept_after_cancel(self, client):
         assert approval.start_at == job_application.hiring_start_at
         assert job_application.state.is_accepted
 
+    @pytest.mark.parametrize(
+        "CRITERIA_KIND,api_returned_payload",
+        [
+            pytest.param(
+                AdministrativeCriteriaKind.RSA,
+                rsa_certified_mocker(),
+                id="rsa",
+            ),
+        ],
+    )
     @freeze_time("2024-09-11")
-    def test_accept_iae__criteria_can_be_certified(self, client, mocker):
-        certify_rsa_mocker = mocker.patch(
+    def test_accept_iae__criteria_can_be_certified(self, CRITERIA_KIND, api_returned_payload, client, mocker):
+        mocked_request = mocker.patch(
             "itou.utils.apis.api_particulier._request",
-            return_value=rsa_certified_mocker(),
+            return_value=api_returned_payload,
         )
         ######### Case 1: if BRSA is one the diagnosis criteria,
         ######### birth place and birth country are required.
         birthdate = datetime.date(1995, 12, 27)
+        # TODO(cms): selecting twice the same criteria cannot happen.
+        # Better iterate over each criterion.
+        diagnosis = IAEEligibilityDiagnosisFactory(
+            job_seeker=self.job_seeker,
+            author_siae=self.company,
+            certifiable=True,
+            criteria_kinds=[CRITERIA_KIND, CRITERIA_KIND],
+        )
```

Comment:
Même remarque ici, mieux vaut créer un state valide.

Reply:
Oui. C'est noté. :)